torch/CMakeLists.txt

```diff
@@ -309,6 +309,11 @@ endif()
 
 
 add_library(torch_python SHARED ${TORCH_PYTHON_SRCS})
+torch_compile_options(torch_python)  # see cmake/public/utils.cmake
+if(NOT WIN32)
+  target_compile_options(torch_python PRIVATE
+      $<$<COMPILE_LANGUAGE:CXX>: -fvisibility=default>)
```

Comment:
What's going on here? Private visibility on torch_python seems doable no?

Reply:
Not feasible, because almost all types in torch_python were previously exported. It takes lots of efforts to mark each type with TORCH_API while still having the risk of breaking extensions. The safest way is to keep the old behavior and change gradually.